bbl_screen-patch/patches/printerui/qml/settings/HardwarePage.qml

```diff
@@ -446,11 +446,11 @@ Rectangle {
             id: toolheadLedButton
             anchors.verticalCenter: toolheadLedTxt.verticalCenter
             anchors.right: buttonGrid.right
-            dynamicChecked: DeviceManager.getSetting("cfw_toolhead_led", false)
+            dynamicChecked: X1Plus.Settings.get("leds.toolhead",false)
             onToggled: {
                 dynamicChecked = checked
```

Comment:
I think this line can die.  then, if it does, magically `x1plus settings put leds.toolhead` will update in the UI.  hmm, but it still wouldn't result in a toggled event...

anyway either way this would not be a regression so definitely +1 from me